content/pages/resources.md

```diff
@@ -0,0 +1,29 @@
+Title: Resources
+Category: pages
+Summary: Feminism101 and Other Resources
+
+# Feminism101 #
```

Comment:
I'd agree with Evy that this would be better with h2 text over h1 (that is, two ##'s)